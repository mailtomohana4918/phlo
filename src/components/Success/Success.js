import React from "react";
import { Box, Typography, Button } from "@material-ui/core";
import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles((theme) => ({
  success: {
    position: "absolute",
    width: "100%",
    height: "100%",
    top: "0",
    left: "0",
    right: "0",
    bottom: "0",
    margin: "auto",
    backgroundColor: "rgba(0,0,0, 0.5)",
  },
  successInner: {
    position: "absolute",
    left: "25%",
    right: "25%",
    top: "50%",
    bottom: "50%",
    margin: "auto",
  },
}));
const Success = (props) => { 
  const classes = useStyles();
  return (
    <div className={classes.success}>
      <div className={classes.successInner}>
        <Box textAlign="center">
          <Typography>
            Booking successful ! You will receive email confirmation
          </Typography>
          <Button type="submit" color="secondary" variant="contained" onClick={props.handleClose}>
            Close
          </Button>
        </Box>
      </div>
    </div>
  );
};

export default Success;
