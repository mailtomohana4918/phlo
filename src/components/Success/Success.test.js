import { render } from "@testing-library/react";
import React from "react";
import Success from "./Success";

describe("Success Component", () => {
  it("renders as expected", () => {
    const { asFragment } = render(<Success />);
    expect(asFragment()).toMatchSnapshot();
  });
});
