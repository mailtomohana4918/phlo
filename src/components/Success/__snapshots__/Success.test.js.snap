// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success Component renders as expected 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-Registerform-1"
  >
    <div
      class="makeStyles-Registerform_inner-2"
    >
      <div
        class="MuiBox-root MuiBox-root-4"
      >
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          Booking successful ! You will receive email confirmation
        </p>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
          tabindex="0"
          type="submit"
        >
          <span
            class="MuiButton-label"
          >
            Close
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
